content/en/tracing/connect_logs_and_traces/dotnet.md

```diff
@@ -115,20 +112,31 @@ For additional examples, see the automatic trace ID injection projects using [NL
 [1]: https://github.com/DataDog/dd-trace-dotnet/blob/master/samples/AutomaticTraceIdInjection/NLog40Example/NLog.config
 [2]: https://github.com/DataDog/dd-trace-dotnet/blob/master/samples/AutomaticTraceIdInjection/NLog45Example/NLog.config
 [3]: https://github.com/DataDog/dd-trace-dotnet/blob/master/samples/AutomaticTraceIdInjection/NLog46Example/NLog.config
+{{% /tab %}}
+{{% tab "Microsoft.Extensions.Logging" %}}
+If you are using `Microsoft.Extensions.Logging`, no additional setup is required. 
```

Comment:
Doesn't the customer also have to enable log scopes for this to work? https://docs.microsoft.com/en-us/aspnet/core/fundamentals/logging/?view=aspnetcore-5.0#log-scopes-1

Reply:
The unfortunate answer is "it depends". It depends entirely on the `ILoggerProvider` implementation. 

The sad reality is there _is_ no file logging provider OOTB (which is why [I wrote one](https://www.nuget.org/packages/NetEscapades.Extensions.Logging.RollingFile) 😄), and as I basically stole the blob storage implementation, mine also requires setting `IncludeScopes=true`.

Which is a long winded way of saying, yes, I think we should add something like:

**Note:** Log injection is only possible with `Microsoft.Extensions.Logging` if your logging provider supports [log scopes](https://docs.microsoft.com/aspnet/core/fundamentals/logging/#log-scopes-1). If log scopes are optional for your provider, be sure to enable them.

Reply:
Updated to add this in https://github.com/DataDog/documentation/pull/11623/commits/18cb25cb6ac91aedf4fadc23007a199a88beca5b